packages/integrations/cloudflare/README.md

```diff
@@ -92,16 +92,18 @@ To do this:
 
 ## Environment Variables
 
-As Cloudflare Pages Functions [provides environment variables differently](https://developers.cloudflare.com/pages/platform/functions/#adding-environment-variables-locally), private environment variables needs to be set through [`vite.define`](https://vitejs.dev/config/shared-options.html#define) to work in builds.
+As Cloudflare Pages Functions [provides environment variables per request](https://developers.cloudflare.com/pages/platform/functions/#adding-environment-variables-locally), private environment variables should only be accessed when a request has happened. Usually, this means moving environment variable access inside a function.
 
 ```js
-// astro.config.mjs
-export default {
-  vite: {
-    define: {
-      'process.env.MY_SECRET': JSON.stringify(process.env.MY_SECRET),
-    },
-  },
+// pages/[id].json.js
+
+export function get({ params }) {
+  // Access environment variables inside a function
```

Comment:
```suggestion
  // Access environment variables per request inside a function
```
Do you think this is helpful? Since we're not showing "Hey don't don't do this anymore!" maybe the extra nugde of "per request" repeated from above will help connect the idea of being in a function? Totally your call.

Otherwise, I'm happy to let you decide whether you think this is better or worse than before. I *do* like that it's more of a complete example, though! So if you think this works, I'd lean towards this now!

Reply:
It does read a bit more succinct. LGTM!